src/main/resources/sbe-schema.xml

```diff
@@ -40,6 +40,7 @@
     <sbe:message name="SessionOpened" id="1" description="Session opened event">
         <field name="sessionId" id="1" type="int64"/>
         <field name="containerId" id="2" type="int32"/>
+        <field name="requestUri" id="3" type="varDataEncoding"/>
```

Comment:
I think this would be better as a fixed-length char array. I'd be happy to pay for the additional padding, since these messages are low volume. This has the added advantage of keeping the messages fixed-length.

Information on fixed-length strings can be found here:
https://github.com/real-logic/simple-binary-encoding/wiki/Java-Users-Guide#fixed-length-string-arrays
https://github.com/real-logic/simple-binary-encoding/blob/master/sbe-samples/src/main/resources/example-schema.xml#L13

When adding new fields to an SBE schema, it is necessary to bump the schema version. Please also tag the new field with the `sinceVersion` attribute, following the guidelines here:
https://github.com/real-logic/simple-binary-encoding/wiki/Message-Versioning#adding-a-new-field